Source/Core/DolphinQt2/Resources.h

```diff
@@ -26,6 +26,11 @@ class Resources final
     LOGO_SMALL
   };
 
+  static QString GetPreferredIconScale(int max = 4);
+  static QPixmap GetScaledIcon(const QString& name, int max = 4,
+                               const QString& dir = QStringLiteral(""));
+  static QPixmap GetScaledThemeIcon(const QString& name, int max = 4);
```

Comment:
Since we control all inputs to these functions, would it be easier to have them take `std::string`s, or at least `const char*`s, and convert internally?